tests/postgres_tests/test_aggregates.py

```diff
@@ -147,7 +147,7 @@ def test_default_argument(self):
                     )
                     self.assertEqual(values, {"aggregation": expected_result})
                 # Empty result when query must be executed.
-                with self.assertNumQueries(1):
+                with transaction.atomic(), self.assertNumQueries(1):
```

Comment:
unneeded?

Reply:
I added it because if any queries failed in the context it made the connection usable which kind of defeats the purpose of using `subTest` in the first place.

Could be worth keeping 🤷

Reply:
Ah yes, that makes absolute sense. Sorry for the noise but I am working myself from file to file -- expect more dumb questions about changes 🫣

@felixxm you probably want that change in an extra commit I assume?

Reply:
> @felixxm you probably want that change in an extra commit I assume?

Yes :wink: Moved to #16353.